src/tsickle.ts

```diff
@@ -2079,6 +2084,56 @@ function addClutzAliases(
   return dtsFileContent;
 }
 
+/**
+ * Returns 2 strings specifying the generic type arguments for the symbol.  The constrained params
+ * include any `T extends foo` arguments, the regular params are just a list of the type symbols,
+ * since we need the constraints on the LHS of the alias declaration, but can't have them on the
+ * RHS.
+ */
+function getGenericTypeParameters(symbol: ts.Symbol): {params: string, constrainedParams: string} {
+  if (!symbol.declarations) {
+    return {params: '', constrainedParams: ''};
+  }
+
+  if (!symbol.declarations[0]) {
+    return {params: '', constrainedParams: ''};
+  }
+
+  const declaration = symbol.declarations[0];
```

Comment:
add a comment about the assumption we talked about here 'all declarations have to have matching generic types'